controllers/index.js

```diff
@@ -0,0 +1,6 @@
+const User = require("./user.controller");
+const Event = require("./event.controller");
+module.exports = {
+  User,
+  Event,
+};
```

Comment:
you can skip this file by adding controller itself in the route file same as you are importing services into the controller

Reply:
Is this way is not fine sir?